mitre/saf
# emasser Docker Build Fix

dependabot was changing the inquire and inquirer-file-tree-selection-prompt

Version above the listed here do not support CommonJs modules.
required fixed versions are:
    "inquirer": "8.0.0",
    "inquirer-file-tree-selection-prompt": "1.0.19"

How to test - ERROR
1. Clone the master repo
2. On the root directory of the repo Build the Docker package
  a.	docker build -t safdockerbuildtest:latest .
4. Run the newly created image:
   a.	docker run s safdockerbuildtest:latest
  
It should fail due to the wrong inquirer-file-tree-selection-prompt being used

How to test -FIX
1. Clone the emass_clientDependencies repo
2. On the root directory of the repo Build the Docker package
  a.	docker build -t safdockerbuildtest:latest .
4. Run the newly created image:
   a.	docker run s safdockerbuildtest:latest

It should run with the previous error 
Additional test run:
docker run safdockerbuildtest:latest emasser version

**NOTE** 
 - Make sure docker is running
- If the build errors executing the yarn command `yarn --frozen-lockfile --production --network-timeout 600000`, comment line 17 on the Dockerfile
- If the build errors executing the `bash jq curl ca-certificates`, comment line 26 on the Dockerfile
